### PR TITLE
feat: add react-native example

### PR DESCRIPTION
**Pull request by vcellu**
- This cleans up the build for react-native
- Updates the Readme.md with instructions on how to run the RN example.

### CODE REVIEW

File: mashup-example/using-build-tool/qlik-sense-saas/main.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ import './style.css';
 
 async function run() {
   const app = await connect({
-    url: '<Qlik tenant url>', // 'xxxx.us.qlik.com',
-    webIntegrationId: '<Qlik web integration id>', // 'xxx-xxxxxxx-xxxxxxxx',
-    appId: '<App id>', // 'xxxx-xxx-xxx-xxx-xxxxxxx',
+    url: 'wastelands.us.qlikcloud.com', // 'xxxx.us.qlik.com',
+    webIntegrationId: 'TGzZjWAv7OK6F4lp0oQhf_7Y-XCK_DtN', // 'xxx-xxxxxxx-xxxxxxxx',
+    appId: '95ab782b-1d7d-4ecc-acaa-4dd17a3841da', // 'xxxx-xxx-xxx-xxx-xxxxxxx',
```
Comment (LiKang6688) [inline mashup-example/using-build-tool/qlik-sense-saas/main.js:10]: I do not think it is good to share those to the public

File: package.json [hunk 2]
```diff
@@ -82,8 +87,7 @@
     "@nebula.js/cli-sense": "2.13.0",
     "@nebula.js/cli-serve": "2.13.0",
     "@playwright/test": "1.23.1",
-    "@qlik/react-native-simple-grid": "1.0.11",
-    "@react-native-async-storage/async-storage": "1.17.7",
+    "@qlik/react-native-simple-grid": "^1.0.11",
```
Comment (LiKang6688) [inline package.json:90]: please use "@qlik/react-native-simple-grid": "1.0.11",

File: src/nebula-hooks/use-extended-theme.native.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { useTheme } from '@nebula.js/stardust';
+
+const useExtendedTheme = () => {
```
Comment (haxxmaxx) [inline src/nebula-hooks/use-extended-theme.native.js:3]: What is the reason that you are not using the non-native version?

File: mashup-example/using-build-tool/qlik-sense-saas/main.js (hunk 1 above)
Comment (vcellu) [reply mashup-example/using-build-tool/qlik-sense-saas/main.js:10]: That's the mashup example.

Comment (vcellu) [reply mashup-example/using-build-tool/qlik-sense-saas/main.js]: removed

File: package.json (hunk 2 above)
Comment (vcellu) [reply package.json]: changed

File: src/nebula-hooks/use-extended-theme.native.js (hunk 3 above)
Comment (vcellu) [reply src/nebula-hooks/use-extended-theme.native.js:3]: The index is off by one. The native one handles it by creating a proxy.

File: README.md [hunk 4]
```diff
@@ -70,6 +70,22 @@ Compress the generated 'sn-table-ext' folder into the 'application/zip' file for
 | :-----------------------------------------------------------------------------: | :--------------------------------------------------------------: | :------------------------------------------------------------------------: |
 | Copy sn-table-ext into https://your-tenant.us.qlikcloud.com/console/extensions/ | Copy sn-table-ext into Qlik Management Console (QMC)->Extensions | Copy sn-table-ext into ..\Users\<UserName>\Documents\Qlik\Sense\Extensions |
 
+## React Native
```
Comment (LiKang6688) [inline README.md:73]: Nice!

File: .vscode/launch.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (LiKang6688) [inline .vscode/launch.json:1]: then people have to use vscode to do the development?

File: README.md [hunk 6]
```diff
@@ -70,6 +70,22 @@ Compress the generated 'sn-table-ext' folder into the 'application/zip' file for
...
+
+### Pre-reqs
```
Comment (haxxmaxx) [inline README.md:75]: We probably want to add that you need to run in vscode

File: src/handle-data.js [hunk 7]
```diff
@@ -50,6 +50,11 @@ export function getColumnInfo(layout, colIndex, columnOrder) {
       sortDirection: directionMap[info.qSortIndicator],
       dataColIdx: colIndex,
       totalInfo: getTotalInfo(isDim, layout, colIndex, numDims, columnOrder),
+      representation: {
```
Comment (haxxmaxx) [inline src/handle-data.js:53]: Is this needed for the example or part of some other feature?

**Comment by haxxmaxx** [comment]
Could you move the minichart code out of this PR? easier to review them as two seperate ones, also for tracking later

File: .vscode/launch.json (hunk 5 above)
Comment (vcellu) [reply .vscode/launch.json:1]: They're not required, but it's a handy thing, so they don't have to create one all the time.

File: src/handle-data.js (hunk 7 above)
Comment (vcellu) [reply src/handle-data.js:53]: it's needed for charts inside the table

File: mashup-example/using-build-tool/qlik-sense-saas/main.js (hunk 1 above)
Comment (haxxmaxx) [reply mashup-example/using-build-tool/qlik-sense-saas/main.js:10]: looks like they are still here

File: README.md (hunk 6 above)
Comment (vcellu) [reply README.md:75]: I removed the .vscode file.

**Pull request closed without merging**